### PR TITLE
[PMG-11] Sorting users in dropdown list

### PR DESCRIPTION
**Pull request by MGotkowski**
Sorting added to API users_data view.

### CODE REVIEW

**Review by tomaszszymanski129 — CHANGES_REQUESTED** [review]
Users are not being sorted correctly in some cases:
image

File: src/presence_analyzer/views.py [hunk 1]
```diff
@@ -113,14 +113,17 @@ def users_data_view():
     """
     data = get_xml_users()
 
-    return [
+    result = [
```
Comment (tomaszszymanski129) [inline src/presence_analyzer/views.py]: Not necessary just like `data` var above, can be all moved to `return [comprehension]`

**Review by tomaszszymanski129 — APPROVED** [review]
In general it's better to handle such things on backend and pass sorting query param if needed, but the ticket didn't specified it and the provided solution works.

TL;DR Approved.
